12-connectivity.tf

```diff
@@ -106,19 +106,6 @@ resource "azurerm_route_table" "route_table" {
   tags                = var.tags
 }
 
-resource "azurerm_route_table" "appgw" {
```

Comment:
this will likely need `moved` blocks in SDS to account for route table location changing